gensim/models/word2vec.py

```diff
@@ -829,15 +829,58 @@ def init_sims(self, replace=False):
             else:
                 self.syn0norm = (self.syn0 / sqrt((self.syn0 ** 2).sum(-1))[..., newaxis]).astype(REAL)
 
-    @staticmethod
-    def log_accuracy(section):
-        correct, incorrect = len(section['correct']), len(section['incorrect'])
-        if correct + incorrect > 0:
-            logger.info("%s: %.1f%% (%i/%i)" %
-                (section['section'], 100.0 * correct / (correct + incorrect),
-                correct, correct + incorrect))
+    def index_by_count(self):
+        """
+        Reorder the vocabulary so that frequent words
+        have low indices
+        """
+
+        i2w = [pair[0] for pair in sorted(iteritems(self.vocab),
+                  key=lambda item: -item[1].count)]
```

Comment:
Wouldn't it be clearer if you put `reverse=True`: `sorted(iteritems(self.vocab), key=lambda item: item[1].count, reverse=True)` ?

It's more verbose but the intention is clear at least